src/api/__specs__/charities-spec.js

```diff
@@ -13,6 +13,7 @@ describe('charities', () => {
   };
 
   beforeEach(() => {
+    callback = sinon.spy();
```

Comment:
I reverted it back to what it was before. I thought about it some more and const is fine. If people expect an object to be immutable they'll learn quickly enough